README.md

```diff
@@ -23,25 +23,13 @@ The repository is split into different directories based on the cloud provider.
 | [ Azure ](./azure/) 	   | Collection of sample projects using our SDK for Azure 	 |
 | [ GCP   ](./gcp/) 	     | Collection of sample projects using our SDK for GCP   	 |
 
-## Build and run the project locally
-
-Each sample project has it's own instructions. Please refer to those for more details.
-
-In most projects you will need these environment variables:
-
-| Environment variable 	| Description                                                                                                	|
-|----------------------	|------------------------------------------------------------------------------------------------------------	|
-| RESOURCE_GROUP_ID    	| The name of the resource group you are working on in Fractal Cloud                                         	|
-| CLIENT_ID            	| Credentials used to authenticate to Fractal. This field is required by the Fractal SDK                     	|
-| CLIENT_SECRET        	| Credentials used to authenticate to Fractal. This field is required by the Fractal SDK                    	|
-| ENVIRONMENT_SHORT_NAME       	| Required to identify the Azure Subscription or the GCP Project on which the Fractal Cloud Agent is active 	|
-
 ### Connect to YanchWare Maven Repository
 
-You can authenticate to YanchWare Packages with Maven by editing your `~/.m2/settings.xml` file to include your personal access token. Create a new `~/.m2/settings.xml` file if one doesn't exist.
-In order to get the credentials to access the packages please contact your administrator.
+A [GitHub Personal Access Token (PAT)]((https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens)) 
```

Comment:
Is it required also on Public repositories?

Reply:
Yes, it is. Quote from [here](https://docs.github.com/en/packages/learn-github-packages/introduction-to-github-packages#authenticating-to-github-packages):

> You need an access token to publish, install, and delete private, internal, and public packages.